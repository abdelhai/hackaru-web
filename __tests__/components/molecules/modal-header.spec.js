import Factory from '@/__tests__/__setups__/factory';
import ModalHeader from '@/components/molecules/modal-header';

describe('ModalHeader', () => {
  it('render correctly', () => {
    const wrapper = new Factory(ModalHeader, {
      slots: {
        default: '<h1>Title</h1>'
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
