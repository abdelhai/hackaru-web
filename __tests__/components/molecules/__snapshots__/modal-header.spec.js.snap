// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalHeader render correctly 1`] = `
<header
  class="modal-header"
>
  <h1>
    Title
  </h1>
</header>
`;
